internal/controllers/topology/cluster/cluster_controller_test.go

```diff
@@ -64,7 +64,6 @@ var (
 
 func TestClusterReconciler_reconcileNewlyCreatedCluster(t *testing.T) {
 	defer utilfeature.SetFeatureGateDuringTest(t, feature.Gates, feature.ClusterTopology, true)()
```

Comment:
Maybe we should also remove these lines since `EXP_CLUSTER_RESOURCE_SET` also defaults to `true`?